pallets/node-authorization/src/tests.rs

```diff
@@ -453,3 +453,10 @@ fn adding_already_connected_connection_should_fail() {
 		);
 	});
 }
+
+#[test]
+fn test_generate_peer_id_invalid_utf8() {
+	let invalid_node_id: NodeId = vec![0xFF, 0xFE, 0xFD];
+	let result = Pallet::<Test>::generate_peer_id(&invalid_node_id);
+	assert_noop!(result, Error::<Test>::InvalidUtf8);
```

Comment:
Make this into `assert_err!`, As we expect it to fail.

Reply:
Done.